apps/web/src/components/Calendar/body/calendar-body.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import { useCalendarContext } from '../calendar-context'
 import CalendarBodyDay from './day/calendar-body-day'
-import CalendarBodyWeek from './week/calendar-body-week'
```

Comment:
agree we dont need this now, but maybe create a ticket to bring it back eventually?